src/scratchtocatrobat/converter/converter.py

```diff
@@ -226,7 +226,7 @@ class _ScratchToCatrobat(object):
         "turnLeft:": catbricks.TurnLeftBrick,
         "heading:": catbricks.PointInDirectionBrick,
         "forward:": catbricks.MoveNStepsBrick,
-        "pointTowards:": catbricks.PointToBrick,
+        "pointTowards:": lambda sprite_name: catbricks.PointToBrick(SpriteFactory().newInstance(SpriteFactory.SPRITE_SINGLE, sprite_name)),
```

Comment:
You would now create a new sprite instance every time there is a PointToBrick.
But the sprite given to the constructor of PointToBrick could already exist at this point.
You'll probably need a register handler. Look in datacontainer if sprite already exists, then use this sprite instance. Otherwise you would need to create a new sprite instance. Somewhere there is a list called upcoming_sprites. It would need to be added to this list, I think.